types/passport-google-oauth20/index.d.ts

```diff
@@ -278,12 +278,13 @@ export interface AuthenticateOptionsGoogle extends passport.AuthenticateOptions
 }
 
 export interface GoogleCallbackParameters {
-    access_token: string;
-    refresh_token?: string | undefined;
+    accessToken: string;
```

Comment:
Please check my comment on the open discussion https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/58005. Feels like a "dangerous" change to me.